tests/src/Kernel/ConfigHandlerTest.php

```diff
@@ -106,7 +106,7 @@ public function testGeneratePatchFile() {
     $config->setData($configData)->save(TRUE);
 
     // Generate patch after configuration change.
-    $data = $configHandler->generatePatchFile(['node'], FALSE);
+    $data = $configHandler->generatePatchFile(['node'], TRUE);
```

Comment:
Can we rename the test function to make it more clear that we are testing the "from active" functionality